dhall/src/Dhall/Pretty/Internal.hs

```diff
@@ -855,6 +858,16 @@ prettyCharacterSet characterSet expression =
     prettyImportExpression (Note _ a) =
         prettyImportExpression a
     prettyImportExpression a0 =
+        prettyRecordCompletionExpression a0
+
+    prettyRecordCompletionExpression :: Pretty a => Expr Src a -> Doc Ann
```

Comment:
For consistency with the parser, I think we should maybe call this `prettyCompletionExpression`.